lib/http-context.js

```diff
@@ -310,10 +314,15 @@ HttpContext.prototype.done = function () {
   if (this.req.query._format) {
     accepts = this.req.query._format.toLowerCase();
   }
-  var dataExists = typeof data !== 'undefined';
 
+  var acceptableMIMEType = accepts;
+  if (acceptableMIMEType) {
+    acceptableMIMEType = acceptableMIMEType.replace(/;\s*charset=[^;]+;*/i, '');
```

Comment:
@tfemiani The returned `accepts` from `this.req.accepts(this.supportedTypes)` will have only one winning media type